symfony/client/messages.en_US.json

```diff
@@ -235,6 +235,21 @@
           "source": "This page is under development",
           "target": "This page is under development",
           "description": ""
+        },
+        "module_not_available": {
+          "source": "403",
+          "target": "403",
+          "description": ""
+        },
```

Comment:
Do we need this

Reply:
instead of hardcoding i rendering the text from this file for language support.